### PR TITLE
Add documentation for specific job planner events

### PR DESCRIPTION
**Pull request by AlexFalessi**

### CODE REVIEW

**Review by tao-xinxiu — APPROVED** [review]

**Review by Hiba-Alili — APPROVED** [review]

**Comment by fviale** [comment]
Retest this please

File: src/docs/user/references/Notification-service.adoc [hunk 1]
```diff
@@ -263,28 +263,57 @@ request will be triggered.
...
+| An association's status has been updated to FAILED due to the previous execution having an error
+| `CANNOT_SUBMIT`
+| Cannot submit
+| ERROR
+| Job Planner was unable to submit the 12 juin 2020 14h15 the workflow Variable_Propagation in the bucket basic-examples from the association 12 to the scheduler
+| <<_glossary_job_planner,*Job Planner*>> was unable to submit the workflow from the bucket to the scheduler
+| `SUBMISSION_POSTPONED`
+| Submission postponed
+| WARNING
 | Association 7 submission planned at vendredi 12 juin 2020 14 h 05 CEST is postponed because job 145 is on-going
```
Comment (fviale) [inline src/docs/user/references/Notification-service.adoc:287]: Can you change the dates in french to english please ...

**Comment by fviale** [comment]
Retest this please

**Review by fviale — APPROVED** [review]
LGTM

**Comment by AlexFalessi** [comment]
Retest this please

**Comment by AlexFalessi** [comment]
Can one of the admins verify this patch?

**Pull request merged**